assets/scss/_header.scss

```diff
@@ -0,0 +1,57 @@
+.header {
+    background-image: url('../images/header-bg-mobile.svg');
+    background-size: 100% auto;
+    background-repeat: no-repeat;
+
+    @include media-queries('tablet') {
+        background-image: url('../images/header-bg.svg');
+    }
+}
+
+.nav__list {
+    min-height: 20vw;
+    display: flex;
+    align-items: center;
+    list-style: none;
+    padding: 0 5vw 5.5vw;
+
+    @include media-queries('tablet') {
+        min-height: 15vw;
+        padding: 0 5vw 4vw;
+    }
+}
+
+.nav__item {
+    padding: 0 2vw 3vw 2vw;
+    text-decoration: none;
+}
+
+.nav__item:first-child {
+    flex-grow: 1;
+}
+
+.nav__logo {
+    font-family: 'Roboto', sans-serif;
+    font-size: 30px;
+    color: white;
+    text-decoration: none;
+
+    @include media-queries('tablet') {
+        font-size: 50px;
+    }
+}
+
+.nav__link {
+    font-family: 'Roboto Condensed', sans-serif;
+    font-size: 18px;
+    color: white;
+    text-decoration: none;
+
+    @include media-queries('tablet') {
+        font-size: 28px;
+    }
+}
+
+img {
```

Comment:
from what I see it will apply to every element on site. Maybe it belongs somewhere in `main` or `_base`